src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerIssueCase.java

```diff
@@ -90,27 +94,54 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
     }
 
     public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
-                                            CaseDetails<CaseData, State> beforeDetails) {
+                                               CaseDetails<CaseData, State> beforeDetails) {
+
         final CaseData data = details.getData();
         final CicCase cicCase = data.getCicCase();
-        String caseNumber = data.getHyphenatedCaseRef();
+        final String caseNumber = data.getHyphenatedCaseRef();
+        final List<String> errors = new ArrayList<>();
 
         if (!isEmpty(cicCase.getNotifyPartySubject())) {
-            caseIssuedNotification.sendToSubject(details.getData(), caseNumber);
+            try {
+                caseIssuedNotification.sendToSubject(details.getData(), caseNumber);
+            } catch (Exception notificationException) {
+                errors.add("Subject");
+            }
         }
         if (!isEmpty(cicCase.getNotifyPartyApplicant())) {
-            caseIssuedNotification.sendToApplicant(details.getData(), caseNumber);
+            try {
+                caseIssuedNotification.sendToApplicant(details.getData(), caseNumber);
+            } catch (Exception notificationException) {
+                errors.add("Applicant");
+            }
         }
         if (!isEmpty(cicCase.getNotifyPartyRepresentative())) {
-            caseIssuedNotification.sendToRepresentative(details.getData(), caseNumber);
+            try {
+                caseIssuedNotification.sendToRepresentative(details.getData(), caseNumber);
+            } catch (Exception notificationException) {
+                errors.add("Representative");
+            }
         }
         if (!isEmpty(cicCase.getNotifyPartyRespondent())) {
-            caseIssuedNotification.sendToRespondent(details.getData(), caseNumber);
+            try {
+                caseIssuedNotification.sendToRespondent(details.getData(), caseNumber);
+            } catch (Exception notificationException) {
+                errors.add("Respondent");
+            }
         }
 
-        return SubmittedCallbackResponse.builder()
-            .confirmationHeader(format("# Case issued %n##  This case has now been issued. %n## %s",
-                MessageUtil.generateSimpleMessage(details.getData().getCicCase())))
-            .build();
+        if (isEmpty(errors)) {
+            return SubmittedCallbackResponse.builder()
+                .confirmationHeader(format("# Case issued %n##  This case has now been issued. %n## %s",
+                    generateSimpleMessage(details.getData().getCicCase())))
+                .build();
+        } else {
+            return SubmittedCallbackResponse.builder()
+                .confirmationHeader(
+                    format("# Issue to respondent notification failed %n## %s %n## Please resend the notification.",
```

Comment:
Instead of "Issue to respondent notification failed", can we change it to "Issue case notification failed". (When we include 'respondent' in the message, it doesn't look like the notification being sent to all parties.